### ISSUE TITLE
dropdown content do not align from right to left

### ISSUE THREAD

**Issue by leonardo015** [Dogfalo/materialize#1841]
The .dropdown-content seems to align only from left to right relatively to its activator(.dropdown-button).
The issue is when the activator is on the right corner of the page or div. Just like this:
![materialize-dropdown-bug2](https://cloud.githubusercontent.com/assets/2482230/9020405/73dd24a4-37e0-11e5-9085-e3b8c9d91263.png)

It would be nice to have an option to align the .dropdown-content from the right to left relatively to its activator

**Comment by dasBIOS** [comment]
I have the exact same problem. 

It looks like that issue was already addressed/solved: #512 https://github.com/Dogfalo/materialize/issues/512
https://github.com/Dogfalo/materialize/commit/679115764057fb56ebc736e326c3f16b9c4fbbf9

I'm trying with parameter alignment: 'right' but it does not work for me.

Anyone solved right dropdown alignment? How?

**Comment by dasBIOS** [comment]
Parameter is not specified in http://materializecss.com/dropdown.html but it is in http://materializecss.link/dropdown.html

**Comment by dasBIOS** [comment]
Can anyone please help?

**Comment by msolters** [comment]
I have this same problem.  Applying

```
align: 'right'
```

has no effect on the rendering of the dropdown content.

**Comment by msolters** [comment]
I looked into the [dropdown.js](https://github.com/Dogfalo/materialize/blob/master/js/dropdown.js) file, and it seems that there was no code that looks for the `.alignment` property.  So it's in the docs but not actually implemented.

I added some logic to handle `alignment: left` and `alignment: right` (see above PR).  I'd recommend pulling it to see if it fixes the issues you guys have been seeing, it seems to work fine for me (Ubuntu Chrome).
